doc/manual/17-configuration.en.md

```diff
@@ -6,14 +6,19 @@ There are two types of configuration files in Barman:
 
 - **global/general configuration**
 - **server configuration**
+- **model configuration**
```

Comment:
[R5](https://github.com/EnterpriseDB/barman/pull/880/commits/a18f2bda214d20bcb14a0b703968c9d4e512cd29#diff-cb88ce549bfbef5a801899de30b88c00cb5f5544fbad2d1841b20cb24a0f20dcR5) should probably be updated to `There are three types of configuration files in Barman:`

Reply:
Applied.